cellxgene_schema_cli/cellxgene_schema/ontology_processing.py

```diff
@@ -0,0 +1,192 @@
+import owlready2
+import yaml
+import urllib.request
+import os
+import gzip
+import json
+
+
+def _dowload_owls(
+    owl_info_yml=os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "ontology_files/owl_info.yml"
```

Comment:
Import project root instead of repeatedly calling `os.path.dirname(os.path.realpath(__file__))`

Reply:
fixed in 7c95ed36a5ba7981c38d2f505920a1134dd034bb